source/optional_features/index.html.md.erb

```diff
@@ -5,22 +5,15 @@ weight: 130
 
 # Optional features
 
-GOV.UK Pay supports some optional features which need configuration:
-
-* [custom branding](/optional_features/custom_branding/#custom-branding-of-payment-pages) of payment pages
-
-* [Welsh language](/optional_features/welsh_language/#welsh-language-payment-pages) of payment pages
-
-* [delayed capture](/optional_features/delayed_capture/#delayed-capture-of-payments) of payments
-
-* [corporate card surcharges](/optional_features/corporate_card_surcharges/#corporate-card-surcharges) added to payments
-
-* [adding custom metadata](/optional_features/custom_metadata) to payments
-
-* [using your own payment failure pages](/optional_features/use_your_own_error_pages) for your service
-
-* [enabling digital wallet transactions](/optional_features/digital_wallets/#digital-wallets) for your service
-
-* [prefilling fields](/optional_features/prefill_user_details/) on the payment page
-
-* [blocking prepaid cards](/optional_features/block_prepaid_cards/#block-prepaid-cards)
+You can use the following optional features with GOV.UK Pay:
+
+* [add corporate card surcharges](/optional_features/corporate_card_surcharges/#corporate-card-surcharges) to payments
+* [add custom branding](/optional_features/custom_branding/#custom-branding-of-payment-pages) to payment pages
+* [add custom metadata](/optional_features/custom_metadata) to payments
+* [block prepaid cards](/optional_features/block_prepaid_cards/#block-prepaid-cards)
+* [delay capturing payments](/optional_features/delayed_capture/#delayed-capture-of-payments)
+* [enable Apple Pay and Google Pay](/optional_features/digital_wallets/#digital-wallets)
+* [prefill fields](/optional_features/prefill_user_details/) on the **Enter card details** page
+* [take payments over the phone ('MOTO' payments)](/optional_features/moto_payments)
```

Comment:
I'd probably say `take Mail Order Telephone Order (MOTO) payments` instead of `take payments over the phone` as we probably will have another "take payments over the phone" feature that uses Twilio (or something similar). What do you think @katiebates ?

Reply:
this conversation still needs resolution. i'm happy if you don't think this line needs to be changed but let me know.

Reply:
I think in this instance it's best to lead with the plainer language version, because I suspect not everyone will be familiar with the term 'MOTO'.
Definitely agree that this language will be confusing if introduce a Twilio-type feature, and we should definitely revisit the description if that happens. But in terms of what's available now, this description feels ok?

Reply:
Ok, let's settle with the plainer language version. For the text "('MOTO' payments)" would it be a good idea to link the 'MOTO' to something like https://www.expertmarket.co.uk/merchant-accounts/moto-merchant-accounts to explain what MOTO is then?
Ok, I think this description feels ok for what we have now. We also have "('MOTO' payments)" which makes it clear.

Reply:
Ah sorry, I haven't expanded the acronym. It should say:
- take payments over the phone (Mail Order Telephone Order ('MOTO') payments)
I'll fix that. (And decide what to do about the double brackets...)

We wouldn't be able to link to that commercial site I'm afraid. But in terms of explanation of MOTO and the benefits, I expect that'll definitely be something Tracy covers in the product page update. (Then these tech docs help devs with the task of actually doing the thing, at the point they already know what MOTO is and that they need to set it up.)